media_submissions.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: Submission Guidelines
+tagline: Project Jupyter Media Submission Guidelines
+permalink: /media_submissions
+---
+
+## Editorial Principles
+Read more about our [editorial principles](https://jupyter.org/governance/charters/MediaStrategyCharter.html) in our charter.
+
+## Content Guidelines
+The Project Jupyter blog is designed for community members to share valuable and educational content about Jupyter. Blog posts can be about various topics, for example how you got started with Jupyter, the latest extension you built around Jupyter, a quick tutorial on how to use a specific feature, and more. This channel is powered by our community members so there are no particular requirements for how the blog post has to be written. 
+
+Here is an example of a potential structure:
+
+- Title
+- Subtitle (max 140 characters) – summary of what the blog post is about
+- Provide one image that represents your project - it can also be a code snippet
+- Introduce your topic
+- Why is this topic important? What are the benefits for the Jupyter community?
+- Do you have an interesting user story or usage example? Include this here.
+- Pick a specific usage example and provide a step-by-step guide so the readers can follow along. Make sure to include visuals or
+  code examples so readers can follow along easily.
+- Thank/acknowledge the people who helped make the work described in the blog post possible. This includes anyone who helped you
+  write the blog post itself.
+- End the article with the forward-looking statement about your project or research
+
+Blog Post Examples:
+- Shorter post: [European Commission Funds Jupyter Bug Bounty Program](https://blog.jupyter.org/european-commission-funds-jupyter-bug-bounty-program-b7b96f9831e6)
+- Longer post: [Generative AI in Jupyter](https://blog.jupyter.org/generative-ai-in-jupyter-3f7174824862)
+- Post using the Jupyter Voice: [JupyterLab is Ready for Users](https://blog.jupyter.org/jupyterlab-is-ready-for-users-5a6f039b8906)
+
+## RESPONSE TIME: Volunteer working group
+Please note that we will vet content against our current editorial review backlog, and follow up with you if there are any issues. Plan ahead. We are primarily a group of volunteers and we will review requests for publication on a weekly basis.
+
+## BLOG: Submission process (blog.jupyter.org)
+- *Create a Medium account*. If you are new to Medium, create an account by following these [instructions](https://help.medium.com/hc/en-us/articles/115004915268-Sign-in-or-sign-up-to-Medium).
+- *Request permissions*. Email the Jupyter Media Strategy working group (JMS)  at jupyter-media-strategy@googlegroups.com  to have your Medium account to be added as the writer of the blog. Once you are added as the authorized writer, you should receive an email that allows you to upload your blog. 
```

Comment:
@blink1073 - I see a typo here. On line 38 instead of this: "to have your Medium account to be added as the writer of the blog."

I want it to say this "with your Medium handle to be added as a writer of the blog."

Reply:
Fixed.